packages/frontend/editor-ui/src/composables/useClipboard.ts

```diff
@@ -12,6 +13,8 @@ export function useClipboard(
 	},
 ) {
 	const { debounce } = useDebounce();
+	const alwaysFalse = computed(() => false);
+	const isInPiPWindow = inject(IsInPiPWindowSymbol, alwaysFalse);
```

Comment:
Any reason for not using `MaybeRefOrGetter<false>` as type and `false`?
```suggestion
	const isInPiPWindow = inject(IsInPiPWindowSymbol, false);
```

You can then use `unref(isInPiPWindow)` to get the value below.

Reply:
Thanks, didn't know this is possible